src/code/z_debug.c

```diff
@@ -106,60 +108,59 @@ void Regs_Init(void) {
     }
 }
 
-// Called when free movement is active.
-void func_8006375C(s32 arg0, s32 arg1, const char* text) {
+// Function is stubbed, name is assumed by `Debug_SetScreenTextColored`, arguments, and usage
+void Debug_SetScreenText(u8 x, u8 y, const char* text) {
 }
```

Comment:
This function is stubbed, so this name is entirely speculation.
But, I think its a fair assumption considering the function right below it, the args supplied to it which look like screen coordinates, and its one usage in z_player. If you change it so that it actually draws, it looks like this in noclip mod
![debugmode](https://user-images.githubusercontent.com/22045047/208368218-7ef9a385-bf15-4cbe-9007-b75b31f1a62d.png)
e: